### ISSUE TITLE
pimatic-openweather  TypeError: Cannot read property 'icon' of undefined

### ISSUE THREAD

**Issue by WimUtTwente** [pimatic/pimatic#826]
```
    2016-03-11 03:42:53
error [pimatic]: A uncaught exception occured: TypeError: Cannot read property 'icon' of undefined
        at /home/pi/pimatic-app/node_modules/pimatic-openweather/node_modules/openweathermap/openweather.js:99:67
        at IncomingMessage.<anonymous> (/home/pi/pimatic-app/node_modules/pimatic-openweather/node_modules/openweathermap/openweather.js:155:16)
        at IncomingMessage.EventEmitter.emit (events.js:117:20)
        at _stream_readable.js:920:16
        at process._tickDomainCallback (node.js:459:13)
     This is most probably a bug in pimatic or in a module, please report it!

**Comment by mwittig** [comment]
I think this is the same issue as https://github.com/pimatic/pimatic/issues/772

@WimUtTwente Do you also get following error right the before the TypeError occurs?
```
A uncaught exception occured: Error: read ECONNRESET
at errnoException (net.js:901:11)
at TCP.onread (net.js:556:19)
This is most probably a bug in pimatic or in a module, please report it!
```

**Comment by mwittig** [comment]
Fixed with pimatic-openweather@0.8.19
